tekton/tasks/qeb-hwt-task.yaml

```diff
@@ -0,0 +1,105 @@
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: qeb-hwt
+  namespace: default
+spec:
+  inputs:
+    resources:
+      - name: git-repo # target path is /workspace otherwise targetPath: specific needs to be included
+        resourceSpec:
+          type: git
+          description: "The input is code from a git repository"
+
+  outputs:
+    resources:
+      - name: justification
+        type: string
```

Comment:
`type: string` ,  i m not aware if tekton resource support this, the resource it support which i know of is git, pull request, image and cluster resource.

Reply:
yes we have to brainstorm, I just started to learn a bit about Tekton :) so what would be more suitable? Don't we want to see the justification of adviser at the end?

Reply:
The output should be projected back to git/pr or if we want to save it in ceph then we should use cluster resource like storage( i would have to check for the syntax of this).

Reply:
In theory we should create a check run, so maybe we add another step in the task that create the check run and there is no actual output.

Reply:
true, i wanted to talk about the flow of the process today, then saw that we are this on the back burner. Anyway as you said, we need a check run, you can remove the output or point the output to git, if there is no actual output.